### DISCUSSION TITLE
1.7.6

### DISCUSSION THREAD

**Discussion by Eutropios** [PyCQA/docformatter#292] [General]
Hi there,

I'm unable to keep docformatter as a pre-commit hook despite the merged commit addressing the language field due to the commit not being a part of a release. Would you mind releasing 1.7.6 with the latest commits to allow for this tool to be used with pre-commit 4.0?

**Comment by nfelt14** [comment]
This should probably be an issue, not a discussion. The official release is still broken with pre-commit.

**Reply by Eutropios** [reply]
Thanks! I'll open one.
